openapi_python_client/parser/responses.py

```diff
@@ -101,6 +118,8 @@ def response_from_data(*, status_code: int, data: Union[oai.Response, oai.Refere
         return Response(status_code=status_code)
     if response_type == "array" and isinstance(schema_data.items, oai.Reference):
         return ListRefResponse(status_code=status_code, reference=Reference.from_ref(schema_data.items.ref),)
+    if response_type == "string" and schema_data.schema_format in {"binary", "base64"}:
```

Comment:
I think you want "byte" instead of "base64" per OpenAPI spec. I don't think base64 is a standard format.
```suggestion
    if response_type == "string" and schema_data.schema_format in {"binary", "byte"}:
```

Reply:
I think its `byte` [for data](https://swagger.io/docs/specification/data-models/data-types/#file) and `base64` [for responses](https://swagger.io/docs/specification/describing-responses/#response-that-returns-a-file)